frontend/src/app/globals.css

```
@@ -108,6 +123,58 @@
+  /* Auth gradient (brand panel) */
+  --auth-gradient-start: #1C3B57;
+  --auth-gradient-mid: #1a3450;
+  --auth-gradient-end: #265e6a;
+
+  /* Auth spacing */
+  --auth-form-gap: 1.25rem;
+  --auth-section-padding: 4rem;
+  --auth-input-height: 2.75rem;
+
+  /* Auth typography */
+  --auth-font-heading: 'Inter', -apple-system, BlinkMacSystemFont, 'SF Pro Display', system-ui, sans-serif;
```

Stylelint (17.4.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)